backend/lib/models/IndividualUser.js

```diff
@@ -44,7 +43,7 @@ const individualUserSchema = new Schema(
       linkedin: String,
       twitter: String,
       website: String,
-    },
+    }
   },
```

Comment:
Minor, we have a Prettier rule to add trailing commas, not sure why it was removed here.